src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private int cargoCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).